src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -150,6 +166,18 @@ public boolean equals(Object other) {
     @Override
     public void sortPersonList(String parameter, boolean isAscending) throws CommandException {
         addressBook.sort(new PersonComparator().getComparator(parameter, isAscending));
+        setSortSettings(new SortSettings(parameter, isAscending));
+    }
+
+    private void resortPersonList(SortSettings sortSettings) {
```

Comment:
not sure if its just me but since "resort" is an actual word, it kind of requires at least a second read in order to understand what it meant. I think maybe `sortPersonListAgain` might be better? or if you prefer re-sort then `reSortPersonList`?